### DISCUSSION TITLE
Change from SQLLite to MySQL

### DISCUSSION THREAD

**Discussion by ThomasIW** [kimai/kimai#2382] [Questions & Answers]
So since the plan is to go live with Kimai soon I thought it would be a good idea to tell Kimai to start using MySQL instead of the SQLLite Database it is using at the moment.  where excatly do I do that?

**Comment by kevinpapst** [comment]
https://www.kimai.org/documentation/installation.html#recommended-setup
`.env`then run install again as documented
